src/Symfony/Component/Form/Extension/Core/DataTransformer/DateTimeToLocalizedStringTransformer.php

```diff
@@ -118,7 +118,11 @@ public function reverseTransform($value)
         // to DST changes
         $dateOnly = $this->isPatternDateOnly();
 
-        $timestamp = $this->getIntlDateFormatter($dateOnly)->parse($value);
+        try {
+            $timestamp = @$this->getIntlDateFormatter($dateOnly)->parse($value);
```

Comment:
can we wrap only the call that needs it? right now, both `getIntlDateFormatter()` and `parse()` are wrapped, but shouldn't only `parse()` be?

Reply:
Sure, I will push a new commit with this change. Thanks for taking a look!